charts/thin/templates/secret/secret.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
Is it necessary to install the secret when `fuse.enabled=false`?

Reply:
I think even it's not used by fuse,  maybe it's used by some other client like hcfs.

Reply:
I think even it's not used by fuse,  maybe it's used by some other client like hcfs.